Airports/EGXW/Basic.txt

```diff
@@ -0,0 +1,3 @@
+Waddington
+N053.09.57.000 W000.31.20.000
+127.350
```

Comment:
This frequency doesn't seem to be applicable here

Reply:
Changed to match the Twr freq